src/Transport/Config/ExtensionMethods/AzureServiceBusTransportExtensions.cs

```diff
@@ -104,6 +104,31 @@ public static void UseNamespaceNamesInsteadOfConnectionStrings(this TransportExt
         }
 
         /// <summary>
+        /// Choose the name of the default namespace.
+        /// </summary>
+        public static void DefaultNamespaceName(this TransportExtensions<AzureServiceBusTransport> transportExtensions, string name )
+        {
+            var settings = transportExtensions.GetSettings();
+            settings.Set(WellKnownConfigurationKeys.Topology.Addressing.DefaultNamespaceName, name);
+        }
+
+        /// <summary>
+        /// Adds a namespace for a specific purpose.
+        /// </summary>
+        public static void AddNamespace(this TransportExtensions<AzureServiceBusTransport> transportExtensions, string name, string connectionString, NamespacePurpose purpose = NamespacePurpose.Partitioning)
```

Comment:
same as below, targetted namespace sends is an edge case, I see no reason why we should make it explicit